spec/sober_swag/reporting/input/converting/date_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+RSpec.describe SoberSwag::Reporting::Input::Converting::Date do
```

Comment:
nit: This spec is missing a `report_on_input` case.

Reply:
e0efb09